mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go.erb

```diff
@@ -4,14 +4,19 @@ package appengine_test
 <% unless version == 'ga' -%>
 
 import (
-	"testing"
+  "log"
+  "strings"
+  "testing"
 
-	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+  "github.com/hashicorp/terraform-plugin-testing/helper/resource"
+  "github.com/hashicorp/terraform-plugin-testing/terraform"
 	"github.com/hashicorp/terraform-provider-google/google/acctest"
 	"github.com/hashicorp/terraform-provider-google/google/envvar"
 )
 
 func TestAccAppEngineFlexibleAppVersion_update(t *testing.T) {
+	// https://github.com/hashicorp/terraform-provider-google/issues/18239
+	acctest.SkipIfVcr(t)
```

Comment:
This should also be skipped in nightlies since it's failing there too (not a problem with VCR only)